### PR TITLE
Add missing <ul> element to index.html template in notmyidea theme

### PR DESCRIPTION
**Pull request by bberberov**

### CODE REVIEW

**Comment by bberberov** [comment]
Is there a problem with this PR?

File: pelican/themes/notmyidea/templates/index.html [hunk 1]
```diff
@@ -50,10 +50,16 @@ <h1><a href="{{ SITEURL }}/{{ article.url }}" rel="bookmark"
...
-<h2>Pages</h2>
+    {% if pages %}
+    <h2>Pages</h2>
+    <ul>
     {% for page in pages %}
         <li><a href="{{ SITEURL }}/{{ page.url }}">{{ page.title }}</a></li>
     {% endfor %}
+    </ul>
+    {% else %}
+    <p>No posts have been made to this blog yet.</p>
```
Comment (justinmayer) [inline pelican/themes/notmyidea/templates/index.html]: Perhaps this wording instead?
> This site currently has no content.

**Comment by justinmayer** [comment]
This PR includes a bit more than just adding the missing `<ul>` tags, but I see how having a "This site currently has no content" message could be informative for the (hopefully rare) cases where folks preview/publish their sites before adding any articles or pages. I made a suggested change to the wording, however, in order to reflect Pelican's more general site-publishing purpose.

**Comment by bberberov** [comment]
I thought since it's in the `index.html` I should mention something about a blog.  But it's already in the "no articles" and "no pages" `if` section so it makes sense.  All I wanted were the `<ul>`s.  The other stuff was for completeness.

**Comment by bberberov** [comment]
I'm not sure why it's failing.  I just changed the plain text.

**Comment by justinmayer** [comment]
@bberberov: It's not your fault — it's mine. I prematurely un-pinned Python-Markdown, which is why you're seeing that error. It can safely be ignored for the moment.

**Comment by justinmayer** [comment]
Hi @bberberov. Thank you for your patience. Would be so kind as to (1) add a `RELEASE.md` file to this pull request as described in the (just-added) [relevant docs](https://docs.getpelican.com/en/latest/contribute.html#using-git-and-github) (third bullet point), and then (2) [rebase & squash your commits](https://github.com/getpelican/pelican/wiki/Git-Tips#squashing-commits)? Thanks again. I look forward to getting your contribution merged! :grin:

**Comment by bberberov** [comment]
ETA, this weekend.

**Comment by justinmayer** [comment]
Hy Boian. Many thanks for the contribution and for your patience. Hoping to get your fix into a release in the near future. 💫

**Pull request merged**